packages/block-library/src/embed/embed-preview.native.js

```diff
@@ -137,4 +140,4 @@ const EmbedPreview = ( {
 	);
 };
 
-export default memo( EmbedPreview );
+export default memo( withPreferredColorScheme( EmbedPreview ) );
```

Comment:
Seems like we're only using `withPreferredColorScheme` to get and pass down `preferredColorScheme` to `SandBox`. Maybe we can instead use `usePreferredColorSchemeStyle` hook in `SandBox` component directly?

Reply:
Yeah although the `usePreferredColorSchemeStyle` hook is meant to be used for picking the right style depending on the light/dark mode status. In this case, I'd rather use `usePreferredColorScheme` that returns the color scheme instead 😃 .

Reply:
This has been addressed in [this commit](https://github.com/WordPress/gutenberg/pull/34207/commits/279c13714ec934a85383ffddc0013d949efca4f7)

Reply:
Do you think it would be possible/better to use the hook directly in `SandBox` and not make any changes in `EmbedPreview` component?

Reply:
Yeah, I initially thought to add it there but since `SandBox` is a generic component I wanted to leave the color scheme calculation open to the parent. However, I agree that looks cleaner to have the hook directly there.

Reply:
@ceyhun I applied your suggestion in [this commit](https://github.com/WordPress/gutenberg/pull/34207/commits/39274fba75b830e52c7cbb3c121db09f12b8838f). I also updated the branch with the last changes from `trunk` and solve conflicts, the PR is ready for another review 🙇 .